src/modules/common/components/bars/topBar/networkName.js

```diff
@@ -1,42 +1,45 @@
-import React from 'react';
+import React, { useMemo, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Dropdown from 'src/theme/Dropdown/dropdown';
+import Icon from 'src/theme/Icon';
 
+import { networkSelected } from 'src/redux/actions';
 import { networkKeys } from '@network/configuration/networks';
 import { getNetworkName } from '@network/utils/getNetwork';
-import Tooltip from 'src/theme/Tooltip';
 import styles from './network.css';
 
-const Network = ({ network, t, token }) => {
-  const networksList = {
-    [networkKeys.mainNet]: t('Mainnet').toLowerCase(),
-    [networkKeys.testNet]: t('Testnet').toLowerCase(),
-    [networkKeys.customNode]: t('Custom node').toLowerCase(),
-  };
+const Network = ({ t }) => {
+  const [showDropdown, setShowDropdown] = useState(false)
+  const network = useSelector(state => state.network);
+  const dispatch = useDispatch()
   const activeNetwork = getNetworkName(network);
```

Comment:
```suggestion
  const activeNetworkName = getNetworkName(network);
```

I think using `activeNetworkName` would be nicer so that anywhere we use it, we know we are using the network's name (which is of type string)